src/diffusers/models/embeddings.py

```diff
@@ -656,3 +657,21 @@ def forward(
             objs = torch.cat([objs_text, objs_image], dim=1)
 
         return objs
+
+
+class ConvEmbed(nn.Module):
```

Comment:
This is too specific to be in general embeddings IMO. Can we move it directly to open_muse_transformer?